### DISCUSSION TITLE
Data Bricks integration

### DISCUSSION THREAD

**Discussion by andrcami** [sqlalchemy/sqlalchemy#11029] [Usage Questions]
I am looking for an ORM that integrates with DataBricks from Python. the idea is to eventually connect this ORM with GraphQL. 

I have achieved the same behaviour with Entity Framework and MS SQL

**Comment by CaselIT** [comment]
Hi,

You can probably start here https://docs.databricks.com/en/dev-tools/sqlalchemy.html

About the graphql part i think there was a library that integrates with sqlalchemy
